packages/node/src/node.ts

```diff
@@ -55,6 +56,21 @@ export class Node {
       `${nodeConfig.STORE_KEY_PREFIX}/${this.signer.address}`
     );
     this.instructionExecutor = new InstructionExecutor(networkContext);
+    this.instructionExecutor.middlewares.add(
```

Comment:
I think there is a `register` method on `InstructionExecutor`

Reply:
Ah right